src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldLikeEventSymbol.cs

```diff
@@ -15,15 +15,15 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// <summary>
     /// This class represents an event declared in source without explicit accessors.
     /// It implicitly has thread safe accessors and an associated field (of the same
-    /// name), unless it does not have an initializer and is either extern or inside
+    /// name), unless it does not have an initializer and is extern, partial, or inside
```

Comment:
>partial [](http://example.com/codeflow?start=69&length=7)

Are we creating a `SourceFieldLikeEventSymbol` for the declaration part of a `partial event` such as below? If so, that seems surprising to me, since there is no field implied.
```csharp
partial class C
{
    partial event System.Action E { add { } remove { } }
    partial event System.Action E;
}
```

Reply:
Yes, we do, but that's the same we do for `extern` events and they have no fields either.

Reply:
Ok, thanks. It feels like this type name is not quite correct then, since this doesn't necessarily represent "field-like" events as defined in the spec. But that is perhaps an existing issue.